logos/src/logos/dbutils/dbmanager.py

```
@@ -0,0 +1,759 @@
+        return None
+
+    def __exec_init(self):
+        with open("./logos/db/init.sql", "r", encoding="utf-8") as file:
+            sql = file.read()
+            for statement in sql.split(";"):
+                stmt = statement.strip()
+                if stmt:
+                    try:
+                        self.session.execute(text(stmt))
+                    except sqlalchemy.exc.ProgrammingError:
+                        pass
```

Ruff (0.11.9)

Use `contextlib.suppress(sqlalchemy.exc.ProgrammingError)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(sqlalchemy.exc.ProgrammingError)`

(SIM105)